README.md

```diff
@@ -13,20 +12,18 @@ The test data is a flat structure with parent ID pointers. You may choose to use
 the data structure as-is, or have a function transform the data structure into
 some other structure, but treat the test data as a given.
 
-- This exercise should not take you more than three hours to complete
-- Feel free to make this component as cool/interesting as you'd like, as long
-  as it meets the technical requirements
-- You may use a JavaScript framework of your choice, or no framework &mdash;
-  the choice is up to you
-- However, you must not rely on a module or library that already provides a
-  similar tree component
-- Your solution should be coded with reusability in mind
-- Submit a working set of code (ZIP, tarball, or link to JSFiddle, Plunker,
-  GitHub pull request, etc.)
+- This exercise should take you a few hours to complete, and you are encouraged to take as much time as you feel is necessary.
+- Feel free to make this component as cool/interesting as you'd like, as long as it meets the technical requirements. This is your opportunity to show off your skills and knowledge.
+- We'd prefer to see usage of a modern component-based JavaScript framework of your choice (React, Vue, Angular, etc.), but we will consider vanilla submissions as well
+- You should utilize ES6+ syntax (or Typescript) targeting modern browsers
+- You should avoid using any runtime dependencies which provide a similar tree-like structure
+- Your solution should be coded with reusability and maintainability in mind. We'd like to see what your code would look like in a production-like environment.
```

Comment:
IMO this is the sort of thing we should _expect_ from a strong programmer, not something we should have to explicitly tell someone to do.  thoughts?

Reply:
I think it helps frame the exercise as a "real world" example of code, as opposed to a one-off toy exercise. Maybe remove the first sentence here, re-word the second sentence a bit?

Reply:
that's fair.  how about "Keep in mind this should reflect what your code would look like in a production-type environment"?